pkg/controller/provider/container/ovirt/collector.go

```diff
@@ -130,6 +131,10 @@ func (r *Collector) HasParity() bool {
 // Test connect/logout.
 func (r *Collector) Test() (status int, err error) {
 	_, status, err = r.client.system()
+	if status != http.StatusOK {
+		err = liberr.New("got status != 200 from oVirt", "status", status))
```

Comment:
@bennyz how about that change in calling liberr.New?

Reply:
ack